packages/grid/x-grid/test/XGrid.test.tsx

```diff
@@ -14,13 +27,65 @@ async function sleep(duration: number) {
 describe('<XGrid />', () => {
   const render = createClientRender();
 
-  it('should resize the width of the columns', async function test() {
+  before(function beforeHook() {
     if (/jsdom/.test(window.navigator.userAgent)) {
-      // TODO: Unclear why this fails. Not important
-      // since a browser test gives us more confidence anyway
+      // Need layouting
       this.skip();
     }
+  });
+
+  describe('keyboard', () => {
+    const KeyboardTest = () => {
+      const data = useData(100, 20);
+      const transformColSizes = (columns) => columns.map((column) => ({ ...column, width: 60 }));
+
+      return (
+        <div style={{ width: 300, height: 300 }}>
+          <XGrid rows={data.rows} columns={transformColSizes(data.columns)} />
+        </div>
+      );
+    };
+
+    it('cell navigation with arrows ', async () => {
+      render(<KeyboardTest />);
+      await sleep(10);
+      document.querySelector('[data-rowindex="0"]').querySelector('[data-colindex="0"]').focus();
+      expect(getActiveCell()).to.equal('0-0');
 
+      fireEvent.keyDown(document.activeElement, { code: 'ArrowRight' });
```

Comment:
@dtassone What do you think we use event.key instead of event.code? It's the property we have been using so far on the main repository. Any reason to go for code?

Reply:
Nope it should be ok to use key

Reply:
I could find https://developers.google.com/web/updates/2016/04/keyboardevent-keys-codes on the topic

Reply:
Also https://codepen.io/denilsonsa/pen/epmoma

> To decide which one you want to use, ask yourself: am I interested in the meaning of the key, or in the physical location? If you are building a game that uses WASD keys, then use .code, as it will work across any keyboard layout. However, if you are interested in detecting if specific inputs were pressed (e.g. arrows, home, end, page up, page down), then use .key. I believe most applications will end up using .key instead of .code.

Reply:
Yes I checked as well, and changing to key will probably fix some issues not raised yet :)

Reply:
Moving it to #132